### PR TITLE
Readme with documentation

### PR DESCRIPTION
**Pull request by oafanasenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
+# API Reference 2.0
 
+##Introduction
```
Comment (LeoSko) [inline README.md]: `##Introduction` won't work, `## Introduction` will

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+The Open API 2.0 is the new version of the publicly available protobuf-based API developed by Spotware. It allows third-party service providers to integrate additional tools and applications for trading and analysis with the data and functionality from Spotware platform by getting all the required cTID data, market data and performing all possible trading operations on behalf of other cTrader users.
+
+##Documentation
```
Comment (LeoSko) [inline README.md]: Same, `## Documentation`

File: README.md [hunk 3]
```diff
@@ -1 +1,62 @@
...
+Full documentation can be found on [Spotware Connect website](https://connect.spotware.com/docs/open_api_2)
+
+## How to start
+1. Download this project.
+2. Import the project to the chosen IDE as a Maven project.
+4. Run command `mvn install` to download libraries and compile Protocol Buffer classes.
```
Comment (LeoSko) [inline README.md]: Let's make numbers either go one after another, or use just `1.` for all of them.

File: README.md [hunk 4]
```diff
@@ -1 +1,62 @@
...
+0. Register your application as described [here](https://connect.spotware.com/docs/open_api_2/getting_started_v2). Set `CLIENT_PUBLIC_ID` and `CLIENT_SECRET`
+ in [application.properties](./openapi2-example/src/main/resources/application.properties). 
+0. Login to [Demo cTrader](https://ct.spotware.com/) to create an account.
+0. Generate a token as decribed [here](https://connect.spotware.com/docs/open_api_2/getting_started_v2/open_authentication_v2). Set `ACCESS_TOKEN` and 
+`CTID_TRADER_ACCOUNT_ID` in [application.properties](./openapi2-example/src/main/resources/application.properties).
```
Comment (LeoSko) [inline README.md]: No need for a new line.

File: README.md [hunk 5]
```diff
@@ -1 +1,62 @@
...
+The Open API 2.0 is the new version of the publicly available protobuf-based API developed by Spotware. It allows third-party service providers to integrate additional tools and applications for trading and analysis with the data and functionality from Spotware platform by getting all the required cTID data, market data and performing all possible trading operations on behalf of other cTrader users.
+
+##Documentation
+Full documentation can be found on [Spotware Connect website](https://connect.spotware.com/docs/open_api_2)
+
+## How to start
+1. Download this project.
+2. Import the project to the chosen IDE as a Maven project.
+4. Run command `mvn install` to download libraries and compile Protocol Buffer classes.
+0. Register your application as described [here](https://connect.spotware.com/docs/open_api_2/getting_started_v2). Set `CLIENT_PUBLIC_ID` and `CLIENT_SECRET`
```
Comment (LeoSko) [inline README.md]: Specify file where to put those parameters.

File: README.md [hunk 6]
```diff
@@ -1 +1,62 @@
...
+5. Run examples from `openapi2-example` submodule.
+
+## Suggested Popular Development Platform IDE
+1. [Eclipse](https://www.eclipse.org/downloads/)
+2. [Spring Tool Suite](https://spring.io/tools3/sts/all)
+3. [IntelliJ IDEA](https://www.jetbrains.com/idea/download/)
+
+## Basic used libraries
+1. Google Protocol Buffers (Protobuf). [Documentation reference](https://connect.spotware.com/docs/open_api_2/introduction_to_protocol_buffers_v2). 
+[Official site](https://developers.google.com/protocol-buffers/).
```
Comment (LeoSko) [inline README.md]: New line.

File: README.md [hunk 7]
```diff
@@ -1 +1,62 @@
...
+- authorization
+- assets list
+- symbols list
+- find symbol by ID
+- subscribe on price spots
+- process Symbol Changed Event
+- get symbol conversion chain
+
+File [application.properties](./openapi2-example/src/main/resources/application.properties) contains properties to be set:
+- `host` You can found it on [Spotware Proxy Cloud page](https://connect.spotware.com/docs/open_api_2/proxy_cloud_v2).
```
Comment (LeoSko) [inline README.md]: > It can be found on

File: README.md [hunk 8]
```diff
@@ -1 +1,62 @@
...
+Default demo host is `demo.ctraderapi.com`
```
Comment (LeoSko) [inline README.md:42]: Let's put an additional link to our "Spotware Proxy Cloud" page in documentation on this line.

**Review by LeoSko — CHANGES_REQUESTED** [review]
Please, review my notes.

File: README.md (hunk 3 above)
Comment (oafanasenko) [reply README.md]: I will use `1`

File: README.md (hunk 4 above)
Comment (oafanasenko) [reply README.md]: IDEA inserts it automatically here

File: README.md (hunk 5 above)
Comment (oafanasenko) [reply README.md]: It is already specified (application.properties)

**Review by LeoSko — APPROVED** [review]

**Pull request merged**